### PR TITLE
[macOS] update macOS 13 template

### PR DESCRIPTION
**Pull request by Alexey-Ayupov**
# Description
Update macOS 13 template

## Check list
- [ ] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/macos/toolsets/toolset-13.json [hunk 1]
```diff
@@ -4,8 +4,7 @@
         "versions": [
             {"link": "14.3", "version": "14.3.0"},
             {"link": "14.2", "version": "14.2.0"},
```
Comment (maxim-lobanov) [inline images/macos/toolsets/toolset-13.json:6]: nit:
```suggestion
            { "link": "14.3", "version": "14.3.0" },
            { "link": "14.2", "version": "14.2.0" },
```

File: images/macos/toolsets/toolset-13.json [hunk 2]
```diff
@@ -112,15 +91,14 @@
...
     },
     "go": {
-        "default": "1.17"
+        "default": "1.18"
     },
     "node": {
-        "default": "16",
+        "default": "18",
         "nvm_versions": [
             "14",
```
Comment (maxim-lobanov) [inline images/macos/toolsets/toolset-13.json:103]: Should we remove `nvm_versions` since we don't install nvm to images?

Comment (maxim-lobanov) [reply images/macos/toolsets/toolset-13.json:103]: The same for `mongodb` and `postgresql`

File: images/macos/toolsets/toolset-13.json (hunk 1 above)
Comment (Alexey-Ayupov) [reply images/macos/toolsets/toolset-13.json]: Fixed

File: images/macos/toolsets/toolset-13.json (hunk 2 above)
Comment (Alexey-Ayupov) [reply images/macos/toolsets/toolset-13.json]: I believe it is ok to remove packages from the list that will not be installed.

**Review by maxim-lobanov — APPROVED** [review]

**Pull request merged**